src/pages/SearchPage.js

```diff
@@ -45,69 +45,68 @@ const defaultProps = {
     reports: {},
 };
 
-class SearchPage extends Component {
-    constructor(props) {
-        super(props);
+function SearchPage(props) {
```

Comment:
Let's use prop destructuring now in order to be aligned with the style guidelines -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring

Reply:
Also, you have to remove usage of `defaultProps` and assign default values during the prop destructuring.